test_scripts/Polices/Validation_of_PolicyTables/233_ATF_preloaded_pt_requesttype_parameters_all_are_invalid.lua

```diff
@@ -40,34 +59,36 @@ function Test:Precondition_stop_sdl()
 end
 
 function Test.TestStep_updatePreloadedPT()
-  local testParameters = {RequestType = {"IVSU","HTML"} }
   commonSteps:DeletePolicyTable()
   commonSteps:DeleteLogsFiles()
-  commonPreconditions:BackupFile(PRELOADED_PT_FILE_NAME)
-  testCasesForPolicySDLErrorsStops.updatePreloadedPT("data.policy_table.app_policies.default", testParameters)
+  update_preloaded_file()
 end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
-function Test:TestStep_checkSdl_Running()
-  --In case SDL stops function will return true
-  local result = testCasesForPolicySDLErrorsStops:CheckSDLShutdown(self)
-  if (result ~= true) then
-    self:FailTestCase("Error: SDL should not stop.")
-  end
+function Test.TestStep_start_sdl()
+  StartSDL(config.pathToSDL, config.ExitOnCrash)
+  os.execute("sleep 5")
+end
+
+function Test:TestStep_checkSdl_Stopped()
+  testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
 end
 
 function Test:TestStep_CheckSDLLogError()
```

Comment:
@istoimenova, Please add check on "BasicCommunication.OnSDLClose" in order to conform to other scripts